src/main/java/com/microsoft/sqlserver/jdbc/SQLServerResultSet.java

```diff
@@ -2373,6 +2373,16 @@ public Object getObject(int columnIndex) throws SQLServerException {
             returnValue = getTime(columnIndex);
         } else if (type == java.sql.Timestamp.class) {
             returnValue = getTimestamp(columnIndex);
+        } else if (type == java.time.LocalDateTime.class) {
+            returnValue = null;
+            java.sql.Timestamp ts = getTimestamp(columnIndex,
+                    Calendar.getInstance(java.util.TimeZone.getTimeZone("UTC")));
```

Comment:
I'm not sure if this is correct given JDBC specifies that the time should be rendered in the JVM default time zone.

Reply:
@mrotteveel The Timestamp needs to be retrieved as UTC to prevent certain date/time values from being corrupted if they are in the "lost hour" when switching from Standard Time to Daylight Time (a.k.a. Summer Time). For example, in my time zone (America/Edmonton), the datetime2 value `2018-03-11T02:00:00` will magically change to `2018-03-11T03:00:00` if we try to retrieve it in the default time zone since there was no `02:00` that morning; the time changed from `01:59` to `03:00`.

Reply:
Clear. You may want to consider skipping timestamp retrieval and instead convert directly from the datetime value to `LocalDateTime`; although that might also be considered as a future improvement.